CHANGES.md

```diff
@@ -111,7 +113,7 @@ Format of the entries must be.
 * Disable the 3DES bulk encryption algorithm for Master Admin Router's TLS. (DCOS_OSS-3537)
 
 
-## DC/OS 1.10.8
+## DC/OS 1.10.6
```

Comment:
It seems like this change was made by mistake. DCOS_OSS-3549 was fixed and merged into 1.10 branch on Jun 21 [1]. DC/OS 1.10.6 was released April 24, 2018 [2].

[1] https://github.com/dcos/dcos/pull/2943
[2] https://docs.mesosphere.com/1.10/release-notes/1.10.6/